providers/slack/src/airflow/providers/slack/transfers/base_sql_to_slack.py

```diff
@@ -21,7 +21,12 @@
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base import BaseHook
-from airflow.models import BaseOperator
+from airflow.providers.slack.version_compat import AIRFLOW_V_3_0_PLUS
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.sdk import BaseOperator
+else:
+    from airflow.models import BaseOperator
```

Comment:
The idea was that the (redundant) selection of where to get BaseOperator from is moved to version_compat.py such that the few lines of boilerplate are in one place. Then you can import directly from there.

Reply:
Ah, no need to be sorry. Was wondering why but this is a good reason :-D